CHANGELOG.next.md

```diff
@@ -36,12 +31,6 @@ Thanks, you're awesome :-) -->
 
 #### Improvements
 
-* ECS scripts now use Python 3.6+. #674
-* schema_reader.py now reliably supports chaining reusable fieldsets together. #722
-* Allow the artifact generator to consider and output only a subset of fields. #737
-* Add support for reusing fields in places other than the top level of the destination fieldset. #739
-* Add support for specifying the directory to write the generated files. #748
-
```

Comment:
Please put back all of the missing lines in the changelog.

Reply:
fixed